backend/src/DataSources/posts.spec.js

```diff
@@ -1,10 +1,22 @@
 const { createTestClient } = require("apollo-server-testing");
 const {gql} = require("apollo-server");
 const {Post, PostsDataSource} = require("./posts-data-source");
+const {User, UsersDataSource} = require("./users-data-source");
+const { GraphQLError } = require('graphql');
 const Server = require("../server");
+const utils = require("../utils");
 
 let postsMemory = new PostsDataSource();
-const server = new Server({dataSources: ()=> ({postsDataSrc: postsMemory})});
+let usersMemory = new UsersDataSource();
+usersMemory.users = [...utils.defaultUsers];
+let decoded
+
+beforeEach(() => {
+    decoded = {}
+})
```

Comment:
```suggestion
let usersMemory
let decoded

beforeEach(() => {
    decoded = {}
    usersMemory = new UsersDataSource();
    usersMemory.users = [...utils.defaultUsers];
})
```
Ensure atomic tests consistently. Maybe you're going to add/delete users in this test?